src/bentoml/_internal/frameworks/easyocr.py

```diff
@@ -0,0 +1,216 @@
+from __future__ import annotations
+
+import typing as t
+import logging
+from types import ModuleType
+
+import bentoml
+
+from ..tag import Tag
+from ..utils.pkg import get_pkg_version
+from ...exceptions import NotFound
+from ...exceptions import MissingDependencyException
+from ..models.model import Model
+from ..models.model import ModelContext
+from ..models.model import ModelOptions
+from ..models.model import ModelSignature
+from .common.pytorch import torch
+from .common.pytorch import PyTorchTensorContainer
+
+try:
+    import easyocr
+except ImportError:  # pragma: no cover
+    raise MissingDependencyException(
+        "'easyocr' is required in order to use module 'bentoml.easyocr'. Install easyocr with 'pip install easyocr'."
+    )
+
+try:
+    import cloudpickle
+except ImportError:  # pragma: no cover
+    raise MissingDependencyException(
```

Comment:
no need to check this, cloudpickle is already a bentoml dependency

Reply:
there might be cases where cloudpickle is uninstalled afterward?

Reply:
That's not a concern since it's required by BentoML.. otherwise, we need to do this for every dependency